### ISSUE TITLE
Visual Studio package crashes after auto-update

### ISSUE THREAD

**Issue by ramayasket** [menees/VsTools#7]
I have installed the version 2022.0.2 more than a year ago, and used it uneventfully since that. After the extension was updated to 2022.0.8 (by the auto-update check box), it caused a visualstudio message that package `Main`_something_ has failed loading, and do I want to keep being notified about that package.

I had to re-install the previous version and disable auto-update, so I can work safely.

I am attaching the ActivityLog.xml from that moment (had to rename it to .txt in order to attach).

[ActivityLog.txt](https://github.com/menees/VsTools/files/13583104/ActivityLog.txt)

**Comment by menees** [comment]
How unfortunate. I updated the VS SDK packages ([here](https://github.com/menees/VsTools/commit/d673c3e431aac60a2232e17749afad611329a3c4#diff-c9825ffb2ea45af24246228a9400da88797ecb6a1d3744b7cdc0e07f8ff2227c)), but I didn't know that would require everyone to be on a VS 17.8.x or later release. I'll go back to the old VS SDKs. Sorry for the inconvenience, and thanks for letting me know.

For anyone else coming in later, here's the relevant error from the end of the [ActivityLog.txt](https://github.com/menees/VsTools/files/13583104/ActivityLog.txt):
``` stacktrace
SetSite failed for package [MainPackage]Source: 'Menees.VsTools' Description: Could not load file or assembly 'Microsoft.VisualStudio.Threading, Version=17.8.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.VisualStudio.Threading, Version=17.8.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
File name: 'Microsoft.VisualStudio.Threading, Version=17.8.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
   at Menees.VsTools.MainPackage.InitializeAsync(CancellationToken cancellationToken, IProgress`1 progress)
   at Microsoft.VisualStudio.Shell.AsyncPackage.<>c__DisplayClass21_0.<<Microsoft-VisualStudio-Shell-Interop-IAsyncLoadablePackageInitialize-Initialize>b__1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.VisualStudio.Services.VsTask.RethrowException(AggregateException e)
   at Microsoft.VisualStudio.Services.VsTask.InternalGetResult(Boolean ignoreUIThreadCheck)

WRN: Assembly binding logging is turned OFF.
To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.
Note: There is some performance penalty associated with assembly bind failure logging.
To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].
```

**Comment by menees** [comment]
This is fixed in the 2022.0.9 release.
